core/bot.py

```
@@ -75,7 +92,73 @@ async def handle_pr_review(message, say, context):
+        #     except Exception as e:
+        #         logger.error(f"Error in SQL bot: {e}")
+        @self.app.message("sql bot")
+        async def handle_sql_bot(message, say, context):
+            trace("slack.sql_bot_request", {
+                "user_id": message['user'],
+                "text": message['text'][:200]
+            })
+            await say("Hello, I am the SQL bot. How can I help you today?")
+            try:
+                text = message['text']
+                user_id = message['user']
```

Ruff (0.12.2)

Local variable `user_id` is assigned to but never used

Remove assignment to unused variable `user_id`

(F841)